gnocchi/carbonara.py

```diff
@@ -331,8 +331,8 @@ def unserialize(cls, data, block_size, back_window):
 
     def serialize(self):
         # NOTE(jd) Use a double delta encoding for timestamps
-        timestamps = numpy.insert(numpy.diff(self.ts.index),
-                                  0, self.first.value)
+        timestamps = numpy.insert(numpy.diff(self.ts.index), 0,
+                                  int(datetime64_to_epoch(self.first) * 10e8))
```

Comment:
`self.first` is already a `timedelta64[ns]`. It seems to work without conversion indeed, I've simplified.